lib/pengine/pe_output.c

```diff
@@ -66,11 +66,33 @@ pe__group_xml(pcmk__output_t *out, va_list args)
     return rc;
 }
 
+static int
+pe__group_text(pcmk__output_t *out, va_list args)
+{
+    long options = va_arg(args, long);
+    resource_t *rsc = va_arg(args, resource_t *);
+
+    GListPtr gIter = rsc->children;
+
+    fprintf(out->dest, "Resource Group: %s", rsc->id);
+    for (; gIter; gIter = gIter->next) {
+        resource_t *child_rsc = (resource_t *) gIter->data;
+
+        out->message(out, crm_element_name(child_rsc->xml), options, child_rsc);
+    }
```

Comment:
We do need to check pe_print_brief and have an equivalent of print_rscs_brief()

Reply:
Fixed.